ocs_ci/ocs/ui/views.py

```diff
@@ -40,6 +40,7 @@
     "search_operator_installed": ('input[data-test-id="item-filter"]', By.CSS_SELECTOR),
     "thin_sc": ('a[id="thin-link"]', By.CSS_SELECTOR),
     "gp2_sc": ('a[id="gp2-link"]', By.CSS_SELECTOR),
+    "standard_sc": ('a[id="standard-link"]', By.CSS_SELECTOR),
```

Comment:
I have a question. Couldn't that be as well `("standard-link", By.ID)`  like some of the other lines you added in this PR? 

It simplifies the readability and I assume that CSS_SELECTOR works harder than finding an element by ID

Reply:
@jimdevops19 I did not try it.